src/app/modules/main/chat_section/module.nim

```diff
@@ -679,9 +679,11 @@ method blockContact*(self: Module, publicKey: string) =
 method onContactBlocked*(self: Module, publicKey: string) =
   self.view.contactRequestsModel().removeItemById(publicKey)
   self.view.chatsModel().blockUnblockItemOrSubItemById(publicKey, blocked=true)
+  self.updateParentBadgeNotifications()
```

Comment:
onContactDetailsUpdated() is not required here ?

Reply:
as I understood no because onContactDetailsUpdated will not do anything here, while during unblocking onContactDetailsUpdated will add deleted item to the contactRequestsModel (it's removing during the block from there)